runtime/rococo/src/bridge_messages.rs

```diff
@@ -0,0 +1,409 @@
+			crate::TransactionByteFee::get(),
+			pallet_transaction_payment::Pallet::<crate::Runtime>::next_fee_multiplier(),
+			|weight| crate::constants::fee::WeightToFee::calc(&weight),
+			transaction,
+		)
+	}
+}
+
+impl<B, GI> TargetHeaderChain<messages_source::FromThisChainMessagePayload<B>, crate::AccountId>
+	for RococoLikeChain<B, GI>
+where
+	B: MessageBridge,
+	B::ThisChain: ChainWithMessages<AccountId = crate::AccountId>,
+	B::BridgedChain: ChainWithMessages<Hash = crate::Hash>,
+	GI: 'static,
+	crate::Runtime: pallet_bridge_grandpa::Config<GI> + pallet_bridge_messages::Config<B::BridgedMessagesInstance>,
+	<<crate::Runtime as pallet_bridge_grandpa::Config<GI>>::BridgedChain as bp_runtime::Chain>::Hash: From<crate::Hash>,
+{
+	type Error = &'static str;
+	type MessagesDeliveryProof = messages_source::FromBridgedChainMessagesDeliveryProof<crate::Hash>;
+
+	fn verify_message(payload: &messages_source::FromThisChainMessagePayload<B>) -> Result<(), Self::Error> {
+		messages_source::verify_chain_message::<B>(payload)
+	}
+
+	fn verify_messages_delivery_proof(
+		proof: Self::MessagesDeliveryProof,
+	) -> Result<(LaneId, InboundLaneData<crate::AccountId>), Self::Error> {
+		messages_source::verify_messages_delivery_proof::<B, crate::Runtime, GI>(proof)
+	}
+}
+
+impl<B, GI> SourceHeaderChain<crate::Balance> for RococoLikeChain<B, GI>
+where
+	B: MessageBridge,
+	B::BridgedChain: ChainWithMessages<Balance = crate::Balance, Hash = crate::Hash>,
+	GI: 'static,
+	crate::Runtime: pallet_bridge_grandpa::Config<GI> + pallet_bridge_messages::Config<B::BridgedMessagesInstance>,
+	<<crate::Runtime as pallet_bridge_grandpa::Config<GI>>::BridgedChain as bp_runtime::Chain>::Hash: From<crate::Hash>,
+{
+	type Error = &'static str;
+	type MessagesProof = messages_target::FromBridgedChainMessagesProof<crate::Hash>;
+
+	fn verify_messages_proof(
+		proof: Self::MessagesProof,
+		messages_count: u32,
+	) -> Result<ProvedMessages<Message<crate::Balance>>, Self::Error> {
+		messages_target::verify_messages_proof::<B, crate::Runtime, GI>(proof, messages_count)
+	}
+}
+
+/// The cost of delivery confirmation transaction.
+pub struct GetDeliveryConfirmationTransactionFee;
+
+impl Get<crate::Balance> for GetDeliveryConfirmationTransactionFee {
+	fn get() -> crate::Balance {
+		<RococoAtRococo as ThisChainWithMessages>::transaction_payment(
+			RococoAtRococo::estimate_delivery_confirmation_transaction()
+		)
+	}
+}
+
+/// This module contains definitions that are used by the messages pallet instance, 'deployed' at Rococo.
+mod at_rococo {
+	use super::*;
+
+	/// Message bridge that is 'deployed' at Rococo chain and connecting it to Wococo chain.
+	#[derive(RuntimeDebug, Clone, Copy)]
+	pub struct AtRococoWithWococoMessageBridge;
+
+	impl MessageBridge for AtRococoWithWococoMessageBridge {
+		const THIS_CHAIN_ID: ChainId = ROCOCO_CHAIN_ID;
+		const BRIDGED_CHAIN_ID: ChainId = WOCOCO_CHAIN_ID;
+		const RELAYER_FEE_PERCENT: u32 = 10;
```

Comment:
I think in the future it would be cool to have relayers dictate this rate themselves in order to create more realistic market dynamics